packages/fork-choice/src/forkChoice/forkChoice.ts

```diff
@@ -1617,6 +1617,7 @@ export class ForkChoice implements IForkChoice {
  * the specs mandates validating terminal conditions on the previously
  * imported merge block.
  */
+// TODO GLOAS: See if we need to update this for gloas
```

Comment:
I don't think we do as this is only called if `isMergeTransitionBlock`, we should consider removing all the merge transition code